.circleci/config.yml

```diff
@@ -71,13 +88,13 @@ jobs:
       - checkout
       - run: 
           name: Build docker image
-          command: docker build -t klaatu:latest -f Dockerfile.desktop .
+          command: docker build -t klaatu:firefox-release -f Dockerfile.desktop .
       - run: 
           name: Run tests
           command: |
             set -ex
             docker compose up -d
-            docker compose run klaatu tox -e bdd-tests -- --reruns 2 --experiment-branch control --variables tests/fixtures/test_experiment.json --run-firefox-release --private-browsing-enabled
+            docker compose run klaatu tox -e bdd-tests -- --reruns 2 --experiment-branch control --variables tests/fixtures/test_experiment.json --private-browsing-enabled
```

Comment:
And I think this was the same as L63 (Nightly) and L80 (Beta), with the exception of this is missing `--run-update-test` flag.